appengine_flexible/helloworld/app.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
```

Comment:
Copyright dates should not be updated on existing files.

Reply:
Undid the copyright updates.